elasticsearch/src/data_frame_transform_deprecated.rs

```diff
@@ -0,0 +1,1388 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// -----------------------------------------------
+// This file is generated, Please do not edit it manually.
+// Run the following in the root of the repo to regenerate:
+//
+// cargo make generate-api
+// -----------------------------------------------
+
+#![cfg(feature = "beta-apis")]
+#![allow(unused_imports)]
+use crate::{
+    client::Elasticsearch,
+    error::Error,
+    http::{
+        headers::{HeaderMap, HeaderName, HeaderValue, ACCEPT, CONTENT_TYPE},
+        request::{Body, JsonBody, NdBody, PARTS_ENCODED},
+        response::Response,
+        transport::Transport,
+        Method,
+    },
+    params::*,
+};
+use percent_encoding::percent_encode;
+use serde::Serialize;
+use std::{borrow::Cow, time::Duration};
+#[cfg(feature = "beta-apis")]
+#[derive(Debug, Clone, PartialEq)]
+#[doc = "API parts for the Data Frame Transform Deprecated Delete Transform API"]
+pub enum DataFrameTransformDeprecatedDeleteTransformParts<'b> {
```

Comment:
The whole `data_frame_transform_deprecated` namespace should never be generated. Looking through, I think it's the only exception.

Reply:
Good point! Looking at the Rest spec for this, It seems we could have a rule that filters out deprecated endpoints that are not stable (this namespace is deprecated as the feature has been implemented slightly differently as stable endpoints). If it proves to require significant changes I'll hard-code the exclusion and open an issue to implement it post-7.11.